lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1267,7 +1269,7 @@ def _relation_data_is_valid(certificates_data: dict) -> bool:
             return False
 
     def _on_relation_changed(self, event: RelationChangedEvent) -> None:
-        """Handler triggerred on relation changed events.
+        """Handler triggered on relation changed events.
```

Comment:
Technically this minor change requires a LIBPATCH bump.
[Currently](https://charmhub.io/tls-certificates-interface/libraries/tls_certificates/source-code) LIBPATCH 11 is on ch so need to bump here LIBPATCH to 12.

Reply:
Done.